backend/scripts/get_pin_metadata.py

```diff
@@ -0,0 +1,91 @@
+"""
+PROBABLY THIS FILE IST MOSTLY DEPRECATED
```

Comment:
Looks like it's mostly the same as get_machines_per_locations.py

Reply:
legacy, removed now